src/framework/midi/internal/fluidsynth.cpp

```diff
@@ -230,28 +230,32 @@ bool FluidSynth::handleEvent(const Event& e)
     }
 
     int ret = FLUID_OK;
-    switch (e.type()) {
-    case EventType::ME_NOTEON: {
-        ret = fluid_synth_noteon(m_fluid->synth, e.channel(), e.note(), e.velocity());
+    int channel = 0;
+    if (e.isChannelVoice()) {
```

Comment:
If the event is not `isChannelVoice`, this is an exception? or in this case, the channel must be 0?